plugins/PrivacyManager/MeasurableSettings.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\PrivacyManager;
+
+use Piwik\Config as PiwikConfig;
+use Piwik\Piwik;
+use Piwik\Settings\Measurable\MeasurableSetting;
+use Piwik\Settings\FieldConfig;
+
+/**
+ * Defines Measurable Settings for PrivacyManager.
+ */
+class MeasurableSettings extends \Piwik\Settings\Measurable\MeasurableSettings
+{
+    /** @var MeasurableSetting */
+    public $forceCookielessTracking;
+
+    protected function init()
+    {
+        $this->forceCookielessTracking = $this->createForceCookielessTrackingSetting();
+        $this->forceCookielessTracking->setIsWritableByCurrentUser(!PiwikConfig::getInstance()->Tracker['force_cookieless_tracking']);
+    }
+
+    private function createForceCookielessTrackingSetting(): MeasurableSetting
+    {
+        return $this->makeSetting('forceCookielessTracking', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
+            $field->title = Piwik::translate('PrivacyManager_ForceCookielessTracking');
+            $field->uiControl = FieldConfig::UI_CONTROL_CHECKBOX;
+            $field->description = Piwik::translate('PrivacyManager_ForceCookielessTrackingDescription');
```

Comment:
Could we check if `matomo.js and piwik.js` is writable and if not, show a message that this feature might not work 100% and that only server side the cookie will be ignored? The check for `writable` could work like in the `TrackerJsCheck` class.

Reply:
In that case we actually don't need to check that, as the website setting has no effect on the matomo.js. Only when set globally the matomo.js would be changed. But I can adjust the description a bit, so it's clear that this setting will only ensure the cookies aren't used server side and it's required to disable cookies in the tracker.